src/shadow-embed.js

```diff
@@ -212,7 +227,12 @@ export function importShadowBody(shadowRoot, body, deep) {
     }
   }
   setStyle(resultBody, 'position', 'relative');
-  shadowRoot.appendChild(resultBody);
+  // DO NOT SUBMIT: Find a solution, this is wrong place. Fix the problem above, the style elements need to be placed as the first child
+  //    For example: getRootNode().firstChild , should return a style element
+  const shadowRootElement = doc.createElement('i-amphtml-shadow-root');
+  shadowRootElement.id = shadowRoot.id;
+  shadowRootElement.appendChild(resultBody);
```

Comment:
Huh, you are correct! I added this because the body originally was not being appended to the correct element. So I added this, and commented out the other path. However, since we switched the css isolation to uses classes instead of id's, the `amp-doc-host` can now handle it. 

Glad this was resolved. Thanks for mentioning this, Done